Code/Framework/AzFramework/AzFramework/Physics/Configuration/RigidBodyConfiguration.cpp

```diff
@@ -104,6 +105,24 @@ namespace AzPhysics
         }
     }
 
+    constexpr AZStd::string_view kinematicDescription =
```

Comment:
I think the coding standards have upper camel case for constant names like this https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#constant-names (e.g. KinematicDescription rather than kinematicDescription).